src/app/page.tsx

```diff
@@ -1,113 +1,120 @@
-import Image from "next/image";
+import Avatar from "@/components/Avatar";
```

Comment:
I will create a TS config file route for these so later on we can just reference these by using `@components` - note to self